src/sentry/conf/server.py

```diff
@@ -276,6 +276,14 @@ def env(key, default="", type=None):
 
 ROOT_URLCONF = "sentry.conf.urls"
 
+# TODO(joshuarli): Django 1.10 introduced this option, which restricts the size of a
+# request body. We have some middleware in sentry.middleware.proxy that sets the
+# Content Length to max uint32 in certain cases related to minidump.
+# Once relay's fully rolled out, that can be deleted.
+# Until then, the safest and easiest thing to do is to disable this check
+# to leave things the way they were with Django <1.9.
+DATA_UPLOAD_MAX_MEMORY_SIZE = None
```

Comment:
Just to make sure I understand, once Relay is rolled out then we can have this check in place again, since we won't need to accept massive request bodies since we won't be accepting minidumps and sourcemaps anymore?

Reply:
Yeah, once relay is rolled out, then we'll want to either remove this to use Django's default value or find a suitable value for us (certainly will be less than 2^32 - 1).

Reply:
The reason this is overridden [here](https://github.com/getsentry/sentry/pull/6416/commits/a5eda7ac18030cfe71bde0334144e314d4f9c39e#diff-c36a4be3be43d9945e58255610e02e7dR64) is only for requests with `Transfer-Encoding: chunked`, since those don't have a content length that can be validated. I don't remember the specifics of this change, but there was some internal validation that blew up otherwise.

What would be the default value of this setting? If it is large enough to allow file uploads and submission of crash report events, we can attempt to set it again. Unfortunately, we don't have a test checked in with chunked transfer encoding (I don't know how to do this with a python request library), but we can run manual tests for this.

There was a way to do this properly in Django 1.8+, but I honestly don't remember any details anymore.

Reply:
It's 2621440 (i.e. 2.5 MB), which might be a little low for file uploads?

I think given the imminent full Relay rollout, it might not be worth the time to investigate this further. We're a little behind the originally estimated schedule for Django 1.10.

Reply:
Sorry, I thought I had approved this already. We definitely have endpoints that receive more than 2.5MB, such as release file uploads, chunk uploads, minidump ingestion, crash report ingestion, and attachment ingestion. We can disable this now to return to old behavior, but there definitely needs to be a solution that configures this limit per endpoint. File and chunk upload will not be moved to Relay.